compose-common.yml

```diff
@@ -29,3 +29,39 @@ base:
     # We enable Python hash randomization to protect Django from certain DoS attacks.
     # See https://docs.djangoproject.com/en/1.8/howto/deployment/checklist/#python-options
     - PYTHONHASHSEED=random
+
+conversionenvironment:
+  environment:
+    - DJANGO_DEBUG=true
+
+    # hardened security compared to default settings.
+    - DJANGO_SECURE_BROWSER_XSS_FILTER=true
+    - DJANGO_SECURE_CONTENT_TYPE_NOSNIFF=true
+    - DJANGO_LOG_LEVEL=ERROR
+
+    # Logging
+    - PYTHONUNBUFFERED=non-empty-string
+    - PYTHONIOENCODING=utf-8
+    - LC_ALL=C.UTF-8
+    - LANG=C.UTF-8
```

Comment:
Do this locale/language settings belong with logging or would they belong in a blank-line-separated block (with or without comment) of their own?